hera_sim/visibilities/vis_cpu.py

```diff
@@ -14,7 +16,7 @@
 class VisCPU(VisibilitySimulator):
 
     def __init__(self, bm_pix=100, real_dtype=np.float32,
-                 complex_dtype=np.complex64, **kwargs):
+                 complex_dtype=np.complex64, c=299792458.0, **kwargs):
```

Comment:
Can't the c constant be hard-coded or loaded from astropy?

Reply:
Importing it from astropy now.

Reply:
Done.